scripts/images/rancher-images.txt

```diff
@@ -7,3 +7,9 @@ docker.io/rancher/rancher-webhook:v0.2.0
 docker.io/rancher/shell:v0.1.10
 docker.io/rancher/system-agent:v0.1.0-suc
 docker.io/rancher/system-upgrade-controller:v0.7.5
+docker.io/rancher/mirrored-directxman12-k8s-prometheus-adapter:v0.8.4
+docker.io/rancher/mirrored-prometheus-node-exporter:v1.1.2
+docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.48.0
+docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.48.0
+docker.io/rancher/mirrored-prometheus-prometheus:v2.27.1
+docker.io/rancher/mirrored-kiwigrid-k8s-sidecar:1.12.2
```

Comment:
Can you check if there are any missing images? AFAIK there should be a grafana image?

Reply:
added via https://github.com/harvester/harvester-installer/compare/10e7f83a6fffc2d05be82fe5ac86e669be0fea71..d8bd2be27583944064aeb158df61a5dec502c17b, thanks